### ISSUE TITLE
Video Short Video Crash on iPad

### ISSUE THREAD

**Issue by gungorbasa** [Yummypets/YPImagePicker#259]
**Describe the bug**
Showing an alert view on iPad causes a crash. To fix it, sourceView needs to be set.

**To Reproduce**
Use the gallery to choose video
There is a time min restriction on the video
UIAlertView crashes

**Expected behavior**
See an alertview that says the video is too short or too long.

**Environment (please complete the following information):**
 - Device: [e.g. iPhone6] iPad
 - OS: [e.g. iOS8.1] 9.3.5 -> Buit all iPads have the issue
 - Xcode Version [e.g. 9.4.1] 10
 - Swift Version [e.g. 4.1] 4.2

**Installation Type**
 - Cocoapods *

**Additional context**
UIAlertView crash caused by not providing the accessory view

**Comment by edwardbeecroft** [comment]
Ditto. Crash report:

`Fatal Exception: NSGenericException
Your application has presented a UIAlertController (<UIAlertController: 0x10299a000>) of style UIAlertControllerStyleActionSheet from YPImagePicker.YPImagePicker (<YPImagePicker.YPImagePicker: 0x102172c00>). The modalPresentationStyle of a UIAlertController with this style is UIModalPresentationPopover. You must provide location information for this popover through the alert controller's popoverPresentationController. You must provide either a sourceView and sourceRect or a barButtonItem. If this information is not known when you present the alert controller, you may provide it in the UIPopoverPresentationControllerDelegate method -prepareForPopoverPresentation.`

TL;DR - The alert controller needs a `sourceView` set.

Article on fixing: https://medium.com/@nickmeehan/actionsheet-popover-on-ipad-in-swift-5768dfa82094
